CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ BUG FIXES:
 * Don't show false update action when import resource with sensitive datasource([#1220](https://github.com/opentofu/opentofu/pull/1220))
 * Fix panic when provisioner source and content are both null ([#1376](https://github.com/opentofu/opentofu/pull/1376))
 * Fix large number will be truncated in plan ([#1382](https://github.com/opentofu/opentofu/pull/1382))
+* Fix S3 backend must have permissions to default 'env:' workspace prefix ([#1445](https://github.com/opentofu/opentofu/pull/1445))
```

Comment:
I'd be tempted to say that the changelog should talk about the positive effect here.

```suggestion
* S3 backend no longer requires to have permissions to use the default 'env:' workspace prefix ([#1445](https://github.com/opentofu/opentofu/pull/1445))
```

Reply:
Thanks! Here it is - 182df0bba9c91852721b41298da29ac44d2e9213